src/object/srv_ec_aggregate.c

```diff
@@ -2315,8 +2314,9 @@ agg_object(daos_handle_t ih, vos_iter_entry_t *entry,
 
 	rc = ds_pool_check_dtx_leader(info->api_pool, &entry->ie_oid,
 				      info->api_pool->sp_map_version, true);
-
 	if (rc == 1 && entry->ie_oid.id_shard >= oca.u.ec.e_k) {
+		D_DEBUG(DB_EPC, "Aggregate EC object "DF_UOID" shard %u\n",
+			DP_UOID(entry->ie_oid), entry->ie_oid.id_shard);
```

Comment:
I had added a log msg at PR5864, but not a problem if your PR landed first I can rebase mine.
BTW, DP_UOID already with id_shard info, so need not with extra id_shard.

Reply:
Ah, my PR landed ahead, you need to rebase it....

Reply:
ah, sure.